src/main/resources/hudson/plugins/jira/listissuesparameter/JiraIssueParameterDefinition/config.jelly

```diff
@@ -25,4 +25,7 @@ limitations under the License.
     <f:entry title="${%JQL Search Query}" field="jiraIssueFilter">
         <f:textbox />
     </f:entry>
+        <f:entry title="${%Alternate Summary Fields}" field="altSummaryFields">
```

Comment:
I guess it would be good to mention they are expected to be comma separated?

Reply:
(and maybe have some "?" help text for this option meaning for the example usecase)

Reply:
Done